tests/api_tests/nodes/test_views.py

```diff
@@ -1675,19 +1677,17 @@ def test_update_project_with_no_title_or_category_formatting(self):
         errors = res.json['errors']
         assert(isinstance(errors, list))
         assert_equal(len(errors), 2)
-        assert('category' in res.json['errors'][0]['meta']['field'])
-        assert('This field is required.' in res.json['errors'][0]['detail'])
-        assert('title' in res.json['errors'][1]['meta']['field'])
-        assert('This field is required.' in res.json['errors'][0]['detail'])
+        errors = res.json['errors']
+        assert_items_equal([errors[0]['source'], errors[1]['source']], [{'pointer': '/data/attributes/category'}, {'pointer': '/data/attributes/title'}])
```

Comment:
Can you add a linebreak between the items of *actual* and *expected*, for readability? (for this and the next line)